src/root.component.tsx

```diff
@@ -1,7 +1,20 @@
 import React from 'react';
-import ReactJoyride from 'react-joyride';
+import ReactJoyride, {type Step} from 'react-joyride';
+import {useAppContext, useDefineAppContext} from "@openmrs/esm-framework"
+import {type TutorialContext} from "./types";
 
 const RootComponent: React.FC = () => {
-  return <ReactJoyride steps={[]} run={false} />;
+
+  const [showTutorial, setShowTutorial] = React.useState(false);
+  const [steps, setSteps] = React.useState([]);
+
+  useDefineAppContext<TutorialContext>('tutorial-context', {
+    showTutorial,
+    steps,
+    setShowTutorial: (showTutorial: boolean) => setShowTutorial(showTutorial),
+    setSteps: (steps: Step[]) => setSteps(steps)
+  });
```

Comment:
@ibacher Is this the right way of using the app context? 

I tried to access the context in `src/tutorial/modal.component.tsx` but it returns an `undefined`

```
const tutorialContext = useAppContext<TutorialContext>('tutorial-context');
```

Reply:
The issue was fixed with https://github.com/openmrs/openmrs-esm-core/pull/1080